lib/connect.c

```diff
@@ -362,9 +363,12 @@ struct eyeballer {
   expire_id timeout_id;              /* ID for Curl_expire() */
   CURLcode result;
   int error;
+  int rewinds;                       /* how often we rewinded the addr list */
```

Comment:
Since you only allow rewinding *once*, couldn't this be a single bit instead?

Reply:
changed